Cargo.toml

```
@@ -58,7 +58,8 @@ uuid = { version = "1.1", features = ["v4"] }
 
 # Opt-out serde and stable_deref_trait features
 # https://github.com/Manishearth/triomphe/pull/5
-triomphe = { version = "0.1.3", default-features = false }
+# 0.1.12 requires Rust 1.76
+triomphe = { version = "<0.1.12", default-features = false }
```

Let's keep the lower bound `v0.1.3`, otherwise `cargo +nightly update -Z minimal-versions` will pick up a too old version `v0.1.0` and `triomphe` will fail to compile.

```suggestion
triomphe = { version = ">=0.1.3, <0.1.12", default-features = false }
```

https://github.com/moka-rs/moka/actions/runs/9290898472/job/25581414759?pr=426

* * *

```console
> Run cargo update -Z minimal-versions
...
Adding triomphe v0.1.0 (latest: v0.1.12)
...
> Run cargo tree --features 'sync, future'
...
├── triomphe v0.1.0
│   ├── nodrop v0.1.8
│   │   └── odds v0.2.12
│   ├── serde v1.0.97
│   └── stable_deref_trait v1.0.0

error[E0599]: `triomphe::Arc<_>` is not an iterator
   --> src/sync_base/key_lock.rs:33:21
    |
33  |         if TrioArc::count(&self.lock) <= 2 {
    |                     ^^^^^ `triomphe::Arc<_>` is not an iterator
    |
   ::: /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/triomphe-0.1.0/lib.rs:80:1
```